platform/admin/admin-configuration-configinstaller/src/main/java/org/codice/ddf/admin/configuration/ConfigurationInstaller.java

```diff
@@ -115,18 +125,32 @@ public void init() throws IOException, InvalidSyntaxException {
     if (configs == null) {
       return;
     }
+    // TODO: Could we miss configurations?
     pidDataMap.putAll(
         Arrays.stream(configs)
             .map(configurationContextFactory::createContext)
             .filter(context -> context.getConfigFile() != null)
             .collect(Collectors.toMap(ConfigurationContext::getServicePid, CachedConfigData::new)));
   }
 
+  @Override
+  public void handleStore(ConfigurationContext context) throws IOException {
+    try {
+      AccessController.doPrivileged(
+          (PrivilegedExceptionAction<Object>)
+              () -> {
+                doHandleStore(context);
+                return null;
+              });
+    } catch (PrivilegedActionException e) {
+      // Only caught if the action through a CHECKED exception, thus it MUST be an IOException.
+      throw new IOException(e);
```

Comment:
✏️ You should extract the cause from the PAE. It's likely already an IOE and wouldn't need to be re-wrapped here.